helpers/tf_cfg.py

```diff
@@ -158,6 +208,16 @@ def check(self):
             msg = 'running clients on a remote host "%s" is not supported' % client_hostname
             raise ConfigError(msg)
 
+    def configure_logger(self):
+        """Configure a logger."""
+        file_handler = logging.FileHandler(
+            self.get("General", "log_file", fallback="tests_log.log")
+        )
+        file_formatter = logging.Formatter("%(asctime)s - %(name)s - %(levelname)s - %(message)s")
+        file_handler.setFormatter(file_formatter)
+
+        self.logger.addHandler(file_handler)
```

Comment:
```suggestion
        date_format = "%y-%m-%d %H:%M:%S"
        file_handler = logging.FileHandler(self.get("General", "log_file"))
        file_handler.setFormatter(
            logging.Formatter(
                fmt="%(asctime)s.%(msecs)03d | %(levelname)s | %(message)s",
                datefmt=date_format
            )
        )

        stream_handler = RichHandler()
        stream_handler.setFormatter(
            logging.Formatter(
                fmt=" | %(message)s",
                datefmt=date_format + ".%f"
            )
        )

        self.logger.addHandler(file_handler)
        self.logger.addHandler(stream_handler)
```

I think we should configure this both handlers here and remove `basicConfig` from 23 line. Also I suggest not to use `%(name)s` because this value always equal `helpers.tf_cfg.py` but it doesn't give any information.

Reply:
Make sense, will be updated.